source/javascripts/controllers/_MainController.js.erb

```diff
@@ -664,8 +664,11 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 
 				appService.getAppConfig();
 
-				requestService.getCurrentUserData().then(function(user) {
-					viewModel.currentUser = user;
+				Promise.all([
```

Comment:
I couldn't find the difference, I guess this is mostly just to match prev implementations 🤷 
```suggestion
				$q.all([
```